codec/av1.md

```diff
@@ -108,12 +106,28 @@ The `Block` timestamp is translated from the __[PresentationTime]__ without the
 
 AV1 stored in Matroska restricts the allowed variations among the `Sequence Header OBUs` contained in the `CodecPrivate` and also in-band in the `Blocks`. The changes are restricted to the changes that are allowed for the `Sequence Header OBUs` of a `CVS`, i.e. the contents of the `Sequence Header OBUs` MUST be bit-identical each time a `Sequence Header OBU` appears except for the contents of __[operating_parameters_info]__. Furthermore the dimensions of all output frames MUST be equal.
 
-The first `Sequence Header OBU` of a `CVS` is stored in the `CodecPrivate` of a `Track`.
 
-If a __[operating_parameters_info]__ exists in a `Sequence Header OBU` then each keyframe `Block` MUST contain a `Sequence Header OBU` before the `Frame Header OBUs`.
+# Seeking and Sequence Header Requirements
+
+`SimpleBlocks` with the `Keyframe` flag set and `Blocks` inside `BlockGroups` that do not contain any `ReferenceBlock` elements constitute the Random Access Points (RAP) of a Matroska `Track`.
+
+Upon random access to a RAP or upon starting playback from the beginning the player/demuxer MUST prepend the `Sequence Header OBU` contained in the `CodecPrivate` to the data of the `Block` where decoding starts after having reverted any potentially applying `ContentCompression`. Afterwards playback proceeds normally including consumption of any `Sequence Header OBUs` that are found in the bitstream.
+
+Seeking to a non-RAP is undefined and discouraged.
+
+A muxer MUST make sure that when using a conformant demuxer/player the correct `Sequence Header OBU` is active both during linear access and also after random access to a RAP. In particular, if a `Sequence Header OBU` whose contents differ from the contents of the `Sequence Header OBU` in the `CodecPrivate` is active during consumption of the first `Frame Header OBU` of a RAP when performing linear access from the beginning, then said `Block` MUST contain said `Sequence Header OBU` in front of the first `Frame Header OBU` so that it is also active when performing random access to said RAP.
+
+A muxer MAY omit (strip away and discard) `Sequence Header OBUs` provided the above criteria are still fulfilled afterwards.
+
+*Notes: a) The contents of two `Sequence Header OBUs` of an AV1 bitstream compliant with the requirements for AV1 tracks in Matroska can only differ if __[decoder_model_present_for_this_op]__ is 1 for some operating point for one (hence every) `Sequence Header OBU`. In particular, if __[timing_info_present]__ is set to 0 for any (hence every) `Sequence Header OBU`, all `Sequence Header OBUs` can be omitted from the bitstream.*

```

Comment:
The [decoder_model_present_for_this_op[ i ]] values are part of the things that must coincide for all the `Sequence Header OBUs` of a `CVS`. So if we only have (or presume to have) a `CVS` and if for one `Sequence Header OBU` all [decoder_model_present_for_this_op[ i ]] are 0 (for all applicable i), then all [decoder_model_present_for_this_op[ i ]] for every `Sequence Header OBU` in the whole `CVS`. This equivalence is behind the "hence every" in the above statement. This equivalence is important because it enables a muxer to know whether there will be any in-band `Sequence Header OBUs` right at the beginning of muxing (after analyzing the first `Sequence Header OBU`).

One could shorten this to: "If a `CVS` contains a `Sequence Header OBU` for which __[decoder_model_present_for_this_op]__ is 0 for all present operating points, then the contents of all `Sequence Header OBUs` contained in this `CVS` are bit-identical to each other and hence can be omitted. This applies in particular if __[timing_info_present]__ is set to 0 in any `Sequence Header OBU`.